web_src/js/index.js

```diff
@@ -2606,6 +2612,8 @@ $(document).ready(async () => {
   initRepoStatusChecker();
   initTemplateSearch();
   initContextPopups();
+  initNotificationsTable();
+  initNotificationCount();
```

Comment:
There is a `Promise.all` block a bit further down that's meant to run async `init*` functions and also `await` their result. I guess it wouldn't matter much if you don't care about the result of these functions, but a un-awaited promise does raise a bit of a red flag here.

Reply:
So it turns out these aren't actually awaits so they don't need to go in there.

Reply:
Yeah, if they are synchronous, init in series is fine.